lib/features/rules/BpmnRules.js

```diff
@@ -1068,11 +1068,11 @@ function canConnectMessageFlow(source, target) {
     return false;
   }
 
-  return (
-    isMessageFlowSource(source) &&
-    isMessageFlowTarget(target) &&
-    !isSameOrganization(source, target)
-  );
+  return isMessageFlowSource(source)
```

Comment:
I'm hesitant to merge this, as this is completely unclear what these additions ensure.